pkg/api/kubegraph/analysis/hpa_test.go

```diff
@@ -0,0 +1,28 @@
+package analysis
+
+import (
+	"testing"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	osgraphtest "github.com/openshift/origin/pkg/api/graph/test"
+)
+
+func TestHPAMissingCPUTargetWarning(t *testing.T) {
+	g, _, err := osgraphtest.BuildGraph("./../../../api/graph/test/hpa-missing-cpu-target.yaml")
```

Comment:
@deads2k based on this discussion, unless I'm misunderstanding why  you need these changes ASAP, let's postpone this PR for a while